v3/soda/mango/mango.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "MF17",
+    "vendorId": "0x3A54",
+    "productId": "0x4F5D",
+    "keycodes": ["qmk_lighting"],
+    "menus": ["qmk_rgb_matrix"],

```

Comment:
Where is the RGB code in QMK?

Reply:
I have pr rgb matrix code to qmk

Reply:
see https://github.com/qmk/qmk_firmware/pull/21385#event-9759338521